src/dynlib/dynlib_linux.c

```diff
@@ -21,13 +22,13 @@ typedef struct _dynlib_context_t
 } dynlib_context_t;
 
 K4A_DECLARE_CONTEXT(dynlib_t, dynlib_context_t);
-static char *generate_file_name(const char *name, uint32_t major_ver, uint32_t minor_ver)
+static char *generate_file_name(const char *name, uint32_t version)
 {
     const char *lib_prefix = "lib";
     const char *lib_suffix = "so";
-    size_t max_buffer_size = strlen(name) + strlen(TOSTRING(DYNLIB_MAX_MAJOR_VERSION)) +
-                             strlen(TOSTRING(DYNLIB_MAX_MINOR_VERSION)) + strlen(".") + strlen(".") +
-                             strlen(lib_suffix) + strlen(lib_prefix) + 1;
+    // Format of the depth engine name is: libdepthengine.so.2.0
```

Comment:
What about making the format less dependent on the current version?

```suggestion
    // Format of the depth engine name is: libdepthengine.so.<K4A_PLUGIN_VERSION>.0
    //                                 ex: libdepthengine.so.2.0
```